.github/workflows/e2e-tests-workflow.yml

```diff
@@ -1,3 +1,5 @@
+# Soon-to-be deprecated way of running integ tests. In the future all tests will be moved to
+# https://github.com/opensearch-project/opensearch-dashboards-functional-test
```

Comment:
After moving to `opensearch-dashboards-functional-test`, we should clone `opensearch-dashboards-functional-test` to local to run E2E test? Suggest update readme once code ready in `opensearch-dashboards-functional-test`

Reply:
@ylwu-amzn yes - and good callout about the README. We will need to update once all tests have moved over there, and once we have removed them from this codebase.